src/shared/components/downloadControls/DownloadControls.css

```diff
@@ -0,0 +1,16 @@
+.menu-item {
+    color:#000000;
+    margin:3px;
+    padding:5px;
+    user-select:none;
+}
+
+.menu-item.menu-item-enabled:hover {
+    cursor:pointer;
+    text-decoration:underline;
+}
+
+.menu-item.menu-item-disabled {
```

Comment:
what's disabled for?

Reply:
disabled is one of the options you can set